experiments/memcached-cat/main.go

```diff
@@ -158,6 +122,12 @@ func main() {
 	numberOfAvailableCacheWays := uint64(maxCacheWaysToAssign + minCacheWaysToAssign)
 	wholeCacheMask := 1<<numberOfAvailableCacheWays - 1
 
+	if sensitivity.ShouldLaunchKubernetesCluster() {
```

Comment:
Flags that define that we should start a cluster are in Sensitivity. I guess we should move them too, right?